project/Version.scala

```diff
@@ -1,7 +1,7 @@
 object Version {
   val Scala212 = "2.12.17"
   val Scala213 = "2.13.8"
-  val Scala3   = "3.2.1"
+  val Scala3   = "3.3.0-RC4"
```

Comment:
This should be removed. There are versions already defined in `BuildHelper.scala`.